### PR TITLE
feat(JavaScript): Add the workflow file for CI check

### PR DESCRIPTION
**Pull request by Shyam-Chen**
If this pull request (PR) pertains to **Chinese-to-English translation**, please confirm that you have read the contribution guidelines and complete the checklist below:

- [ ] This PR represents the translation of a single, complete document, or contains only bug fixes.
- [ ] The translation accurately conveys the original meaning and intent of the Chinese version. If deviations exist, I have provided explanatory comments to clarify the reasons.

If this pull request (PR) is associated with **coding or code transpilation**, please attach the relevant console outputs to the PR and complete the following checklist:

- [ ] I have thoroughly reviewed the code, focusing on its formatting, comments, indentation, and file headers.
- [ ] I have confirmed that the code execution outputs are consistent with those produced by the reference code (Python or Java).
- [ ] The code is designed to be compatible on standard operating systems, including Windows, macOS, and Ubuntu.

### CODE REVIEW

File: codes/javascript/test_all.ts [hunk 1]
```diff
@@ -0,0 +1,60 @@
+import { bold, brightRed } from 'jsr:@std/fmt/colors';
```
Comment (krahets) [inline codes/javascript/test_all.ts:1]: Can we use a `.js` file instead? It's a little confusing to test JS code with a `.ts` file.

**Review by krahets — COMMENTED** [review]
Thanks for the PR! Please address the comment

File: .github/workflows/javascript.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 20.x
+      - uses: denoland/setup-deno@v1
```
Comment (krahets) [inline .github/workflows/javascript.yml:23]: Do we need deno for JS?

Comment (Shyam-Chen) [reply .github/workflows/javascript.yml:23]: Yes, `test_all.js` is executed through Deno.

**Review by krahets — APPROVED** [review]
Thanks for the PR!

**Pull request merged**